### ISSUE TITLE
TwitchNoSub 0.7.1 - requesting info about a new update

### ISSUE THREAD

**Issue by OrionGlow** [besuper/TwitchNoSub#116]
good morning Dev.

I'm asking if there are any updates on a possible new version. Version 0.7.1 stopped working shortly after its release due to yet another change by Twitch.
I understand that this is a hobby project, so I'm just asking if a new release is planned, or if it's a "closed" project.
Thank you.

**Comment by Endymi0n74** [comment]
on which browser? os? because i can confirm it works

**Comment by OrionGlow** [comment]
> on which browser? os? because i can confirm it works

Okay... I apologize, you’re right, they do work.... but I swear, they didn't work for me for weeks; out of nowhere, I kept getting subscription requests, to the point where I decided to disable the addon while waiting for an update.
If I hadn't opened a post and then read your reply, I would still have them disabled!
The browsers I use are Brave version 1.67.116 and Opera GX 109.0.5097.130.
I've reactivated the addon on both browsers before replying to you.... and yes, it works again!
Thanks a lot and sorry for the trouble.
